src/test/kotlin/com/hypto/iam/server/apis/OrganizationApiKtTest.kt

```diff
@@ -75,13 +78,67 @@ internal class OrganizationApiKtTest : AbstractContainerBaseTest() {
                 setBody(gson.toJson(requestBody))
             }
             val responseBody = gson.fromJson(response.bodyAsText(), CreateOrganizationResponse::class.java)
+
+            // Assert API response
             assertEquals(HttpStatusCode.Created, response.status)
             assertEquals(ContentType.Application.Json.withCharset(UTF_8), response.contentType())
 
             orgId = responseBody.organization.id
             assertEquals(requestBody.name, responseBody.organization.name)
             assertEquals(10, responseBody.organization.id.length)
 
+            // Assert root user creation
+            val listUsersResponse = client.get("/organizations/$orgId/users") {
+                header(HttpHeaders.ContentType, ContentType.Application.Json.toString())
+                header(HttpHeaders.Authorization, "Bearer ${responseBody.rootUserToken}")
+            }
+            Assertions.assertEquals(HttpStatusCode.OK, listUsersResponse.status)
+            Assertions.assertEquals(
+                ContentType.Application.Json.withCharset(UTF_8),
+                listUsersResponse.contentType()
+            )
+
+            val listUserResponse = gson.fromJson(listUsersResponse.bodyAsText(), UserPaginatedResponse::class.java)
+            Assertions.assertEquals(listUserResponse.data!!.size, 1)
+
+            // Assert policy creation
+            val actResponse = client.get(
+                "/organizations/$orgId/policies?pageSize=50"
+            ) {
+                header(HttpHeaders.ContentType, ContentType.Application.Json.toString())
+                header(
+                    HttpHeaders.Authorization,
+                    "Bearer ${responseBody.rootUserToken}"
+                )
+            }
+            Assertions.assertEquals(HttpStatusCode.OK, actResponse.status)
+            Assertions.assertEquals(
+                ContentType.Application.Json.withCharset(UTF_8),
+                actResponse.contentType()
+            )
+
+            val listPoliciesResponse = gson.fromJson(actResponse.bodyAsText(), PolicyPaginatedResponse::class.java)
+            Assertions.assertNotNull(listPoliciesResponse.nextToken)
+            Assertions.assertEquals(1, listPoliciesResponse.data?.size)
+            Assertions.assertEquals("admin", listPoliciesResponse.data!![0].name)
```

Comment:
Along with the name, can we also check the content so that we can be sure that mustache variable substitution was successful.